Makefile

```
@@ -135,3 +135,9 @@ cask: zip  ## Generate the font cask file (requires Homebrew)
 ifeq ($(UNAME),Darwin)
 	@${FONTCASKER} ${BUILD_DIR}/3270_fonts_$(shell git rev-parse --short HEAD).zip
 endif
+
+image: # Builds the container image
```

```suggestion
image: ## Builds the container image
```

Using two #"s makes it appear in the help